src/include/common/config.h

```diff
@@ -33,12 +33,14 @@ extern std::chrono::duration<int64_t> log_timeout;
 static constexpr int INVALID_PAGE_ID = -1;                                           // invalid page id
 static constexpr int INVALID_TXN_ID = -1;                                            // invalid transaction id
 static constexpr int INVALID_LSN = -1;                                               // invalid log sequence number
-static constexpr int HEADER_PAGE_ID = 0;                                             // the header page id
 static constexpr int BUSTUB_PAGE_SIZE = 4096;                                        // size of a data page in byte
 static constexpr int BUFFER_POOL_SIZE = 10;                                          // size of buffer pool
 static constexpr int LOG_BUFFER_SIZE = ((BUFFER_POOL_SIZE + 1) * BUSTUB_PAGE_SIZE);  // size of a log buffer in byte
 static constexpr int BUCKET_SIZE = 50;                                               // size of extendible hash bucket
-static constexpr int LRUK_REPLACER_K = 10;  // lookback window for lru-k replacer
+static constexpr int LRUK_REPLACER_K = 10;  // default lookback window for lru-k replacer
+
+// TODO(cjtsui) what is this HEADER_PAGE_ID???
+static constexpr int HEADER_PAGE_ID = 0;  // the header page id
```

Comment:
Seems that this is only used in tests... Should consider refactoring

Reply:
#736